### DISCUSSION TITLE
How to "re-add the bulbs via the UI?"

### DISCUSSION THREAD

**Discussion by Thynix** [sbidy/wiz_light#225] [Q&A] [answered]
I updated to 0.4.5b, and the changelog includes

> **YAML Config Support is no longer supported!.** Please re-add the bulbs via the UI.

I haven't been able to find where this is referring to. Any idea what I'm missing?

**Comment by Pixeldk** [comment]
Allso looking for the UI. allmost sure i am an idiot, and blind.

**Comment by Thynix** [comment, marked as answer]
Turns out this means adding WiZ Light under Configuration > Integrations. It'll prompt for bulb name and IP/hostname.
